src/marbl_settings_mod.F90

```diff
@@ -134,7 +134,7 @@ module marbl_settings_mod
       f_graze_sp_poc_lim    = 0.36_r8,  &
       f_photosp_CaCO3       = 0.40_r8,  & ! proportionality between small phyto
                                           !    production and CaCO3 production
-      f_graze_CaCO3_remin   = 0.33_r8,  & ! fraction of spCaCO3 grazing which is remin
```

Comment:
I assume this line will change answers in the default POP setup? Maybe we should move `f_graze_CaCO3_remin` to the settings file with a default of 0.33 and then part of running with coccolithophores would be changing the value to 0.7 in `user_nl_marbl`

Reply:
Yes, I think that would be a good idea.